coffeecard/CoffeeCard.WebApi/Pages/Recover.cshtml.cs

```diff
@@ -25,9 +26,16 @@ public Recover(IAccountService accountService, ITokenService tokenService)
 
         public async Task<IActionResult> OnGet()
         {
-            IsTokenValid = await _tokenService.ValidateToken(Token);
+            try
+            {
+                IsTokenValid = await _tokenService.ValidateTokenIsUnusedAsync(Token);
 
-            if (IsTokenValid) return Page();
+                if (IsTokenValid) return Page();
+            }
+            catch (Exception e)
+            {
+                // ignored
+            }

```

Comment:
This is not an optimal pattern. Why do we catch the Exception? If at all, we must at least log the exception else we would never know the exception was thrown

Reply:
I'll add a log statement. We catch for the same reason as the two above